ruby-challenges.rb

```diff
@@ -23,12 +38,36 @@
 # Expected output: 'bby Rd'
 
 
+beatles_album1 = 'Rubber Soul'
+get_vowels = beatles_album1.delete('aeiou', '')
+
+def my_vowels param
+  p  param.delete 'aeiouAEIOU'
+end
 
-# -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
+my_vowels beatles_album1
+my_vowels beatles_album2
+my_vowels beatles_album3
+
+
+
+#  Question3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
 
 palindrome_tester1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 palindrome_tester2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindromeStrings string
+  if string.downcase == string.reverse.downcase
+    p 'is a palindrome'
+  else
+    p 'is not a palindrome'
+  end
+end
+
+palindromeStrings palindrome_tester1
+palindromeStrings palindrome_tester2
+palindromeStrings palindrome_tester3
```

Comment:
These are great examples of writing out functions once and invoking it multiple times!